src/startupscriptgenerator/src/php/scriptgenerator.go

```diff
@@ -25,12 +27,25 @@ func (gen *PhpStartupScriptGenerator) GenerateEntrypointScript() string {
 
 	logger.LogInformation("Generating script for source at '%s'", gen.SourcePath)
 
+	var _phpOrigin = os.Getenv(consts.PhpOriginEnvVarName)
```

Comment:
do we need our own env var for that? can we / should we ask App Service to export some variable FOR us?

Reply:
there can be multiple ways, at this moment I am just trying to create a foundation image for app service team, once they are happy with it, we can accommodate these changes.